### PR TITLE
DFR-761-Consented-and-Contested-BS-allow-civil-partnership applications

### PR DESCRIPTION
**Pull request by MindGames2015**
…-newPaperCase and another json object to CaseEventToFields-newPaperCase.json with id civilPartnership-newPaperCase

**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

https://tools.hmcts.net/jira/browse/DFR-761

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by EllisD-B — DISMISSED** [review]

**Review by rishikrsharma — CHANGES_REQUESTED** [review]
This change not needed.

File: yarn-audit-known-issues [hunk 1]
```diff
@@ -1 +1 @@
...
...specific inputs\n* noticeable slowdown is observed with inputs above 10k characters\n* users who pass user-provided strings without sanity length checks to moment constructor are vulnerable to (Re)DoS attacks\n\n### Patches\nThe problem is patched in 2.29.4, the patch can be applied to all affected versions with minimal tweaking.\n\n### Workarounds\nIn general, given the proliferation of ReDoS attacks, it makes sense to limit the length of the user input to something sane, like 200 characters or less. I haven't seen legitimate cases of date-time strings longer than that, so all moment users who do pass a user-originating string to constructor are encouraged to apply such a rudimentary filter, that would help with this but also most future ReDoS vulnerabilities.\n\n### References\nThere is an excellent writeup of the issue here: https://github.com/moment/moment/pull/6015#issuecomment-1152961973=\n\n### Details\nThe issue is rooted in the code that removes legacy comments (stuff inside parenthesis) from strings during rfc2822 parsing. `moment(\"(\".repeat(500000))` will take a few minutes to process, which is unacceptable.","url":"https://github.com/advisories/GHSA-wc69-rhjr-hc9g"}}}
```
Comment (rishikrsharma) [inline yarn-audit-known-issues:1]: not needed

File: definitions/contested/json/CaseEventToFields/CaseEventToFields-newPaperCase.json [hunk 2]
```diff
@@ -233,8 +233,20 @@
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "FinancialRemedyContested",
     "CaseEventID": "FR_newPaperCase",
-    "CaseFieldID": "dateOfMarriage",
+    "CaseFieldID": "civilPartnership",
```
[the remark this answers is not in the document]
Comment (MindGames2015) [reply definitions/contested/json/CaseEventToFields/CaseEventToFields-newPaperCase.json]: [This ](https://github.com/hmcts/finrem-case-orchestration-service/pull/770/files#diff-f4167e3b25b9e7c99ba1f1dd6676a16e48c6916955caa92d02a644b21ca38677R121)
Here is the pr Rishi with callback change, that defaults to no. Thanks

File: yarn-audit-known-issues (hunk 1 above)
Comment (MindGames2015) [reply yarn-audit-known-issues:1]: This was showing as a merge conflict Rishi, so I corrected this. Thanks.

**Pull request closed without merging**